flaky test: `chainerx_tests/unit_tests/routines_tests/test_linalg.py::test_PseudoInverse`

https://travis-ci.org/chainer/chainer/jobs/574551735

Occurred in #7783

>`FAIL tests/chainerx_tests/unit_tests/routines_tests/test_linalg.py::test_PseudoInverse_param_3_{rcond=1e-15, shape=(6, 6), dtype='float32'}[native:0]`

```
E       chainer.testing.function_link.FunctionTestError: Outputs do not match the expected values.
E       Indices of outputs that do not match: 0
E       Expected shapes and dtypes: (6, 6):float32
E       Actual shapes and dtypes:   (6, 6):float32

:

E       assert_allclose failed: 
E         shape: (6, 6) (6, 6)
E         dtype: float32 float32
E         i: (3, 5)
E         x[i]: -0.002727508544921875
E         y[i]: -0.002660859376192093
E         relative error[i]: 0.025047985836863518
E         absolute error[i]: 6.66491687297821e-05
E         relative tolerance * |y[i]|: 2.660859376192093e-07
E         absolute tolerance: 1e-05
E         total tolerance: 1.026608593761921e-05
```

--- a/tests/chainerx_tests/unit_tests/routines_tests/test_linalg.py
+++ b/tests/chainerx_tests/unit_tests/routines_tests/test_linalg.py
@@ -332,6 +332,7 @@ class TestPseudoInverse(NumpyLinalgOpTest):
 
     def generate_inputs(self):
         a = numpy.random.random(self.shape).astype(self.dtype)
+        a = a * 10 + numpy.ones(self.shape)
         return a,
 
     def forward_xp(self, inputs, xp):

